packages/cra-external-template/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "package": {
+    "dependencies": {
+      "@reapit/cognito-auth": "^2.1.7",
```

Comment:
Can you do a review of the dependencies here. Feels like we can bin quite a few of them, want to keep it as lean as possible